packages/debug/src/browser/debug-session-contribution.ts

```diff
@@ -114,6 +113,8 @@ export class DefaultDebugSessionFactory implements DebugSessionFactory {
     protected readonly debugPreferences: DebugPreferences;
     @inject(FileService)
     protected readonly fileService: FileService;
+    @inject(ContributionProvider) @named(DebugContribution)
+    protected readonly debugCP: ContributionProvider<DebugContribution>;
```

Comment:
I believe we should be clear in the naming of the variables:

```suggestion
    protected readonly debugContribution: ContributionProvider<DebugContribution>;
```

The same is true for all other occurrences of `debugCP` across the pull-request.

Reply:
What would you like it to be named? it should be `debugContributionProvider` but it seems long to me

Reply:
renamed as `debugContributionProvider`